src/pages/restaurant/reducers/helper.js

```diff
@@ -0,0 +1,155 @@
+
```

Comment:
this is a constant file, should be just under the restaurant folder like we have store.js

Reply:
menuItemsData =>  should be an ENUM